Helper/Config.php

```diff
@@ -559,25 +557,12 @@ public function getConfigurationMode(int $storeId): string
         );
     }
 
-    /**
-     * Returns global configuration values
-     *
-     * @param $field
-     * @param null|int|string $storeId
-     * @return mixed
-     */
-    public function getAdyenAbstractConfigData($field, $storeId = null)
+    public function getAdyenAbstractConfigData($field, int|string $storeId = null): mixed
```

Comment:
Type of the `storeId` argument should be `int` and `field` argument might be string.